openslides_backend/action/actions/poll/stop.py

```diff
@@ -65,18 +66,7 @@ def prefetch(self, action_data: ActionData) -> None:
                 ["user_ids"],
             ),
         ]
-        if self.user_id:
-            requests.append(
-                GetManyRequest(
-                    Collection("user"),
-                    [self.user_id],
-                    [
-                        "organization_management_level",
-                        *[f"group_${meeting_id}_ids" for meeting_id in meeting_ids],
-                    ],
-                )
-            )
-        self.datastore.get_many(requests)
+        self.datastore.get_many(requests, use_changed_models=False)
```

Comment:
I don't know why, but removing this `get` from prefetch, reduces the time for the write in the datastore-service by roundabout  33%.

profile method with locked fields from request user in prefetch/removed from prefetch, poll with 100 users:
/app/openslides_backend/services/datastore/http_engine.py:37(retrieve) tottime: 0.000/0.000    cumtime:0.455/0.304

The difference are the 2 locked fields  user.group_$meeting_ids and user.organization_management_level. 
Could you have a look into the datastore, why the impact of these 2 fields is so big? The percentage remains constant, even with 2000 votes intest.